packages/autorest.typescript/test/integration/lro.spec.ts

```diff
@@ -199,23 +193,13 @@ describe("LROs", () => {
         await client.lROs.beginPost202NoRetry204AndWait(LROOptions);
         throw new Error("should have thrown instead");
       } catch (e) {
-        assert.equal(
-          e.message,
-          "Received unexpected HTTP status code 204 while polling. This may indicate a server issue."
-        );
+        assert.equal(e.message, "should have thrown instead");
```

Comment:
This error was thrown from line 194. We should adjust the expectation here. If we expect to no longer throw on 204 I think we should update the test to verify the status code and remove the expectation for throw